indigo_app/forms.py

```diff
@@ -272,19 +272,25 @@ class WorkFilterForm(forms.Form):
             .order_by('vocabulary__title', 'level_1', 'level_2'))
 
     advanced_filters = ['assent', 'publication', 'repeal', 'amendment', 'commencement',
-                        'primary_subsidiary', 'taxonomies', 'stub', 'completeness']
+                        'primary_subsidiary', 'taxonomies', 'completeness', 'status']
 
     def __init__(self, country, *args, **kwargs):
         self.country = country
         super(WorkFilterForm, self).__init__(*args, **kwargs)
-        self.fields['subtype'] = forms.ChoiceField(choices=[('', 'All works'), ('acts_only', 'Acts only')] + [(s.abbreviation, s.name) for s in Subtype.objects.all()])
+        self.fields['subtype'] = forms.ChoiceField(choices=[('', 'All types'), ('acts_only', 'Act')] + [(s.abbreviation, s.name) for s in Subtype.objects.all()])
```

Comment:
I realise this is a bit of an edge case, but just for the record I had it as 'Acts only' instead of 'Act' for when someone adds 'Act' as a subtype (which did happen with one installation).

Reply:
```suggestion
        self.fields['subtype'] = forms.ChoiceField(choices=[('', 'All types'), ('acts_only', 'Acts only')] + [(s.abbreviation, s.name) for s in Subtype.objects.all()])
```